src/NBitcoin.Tests/ChainTests.cs

```diff
@@ -187,7 +187,7 @@ public void CanFindFork()
 
         private void AssertFork(ConcurrentChain chain, ConcurrentChain chain2, ChainedHeader expectedFork)
         {
-            ChainedHeader fork = this.FindFork(chain, chain2);
+            ChainedHeader fork = FindFork(chain, chain2);
```

Comment:
hmm I wonder why this. has been removed from here and other places.

Reply:
Hm, that's really odd. 

I've saw resharper chocking on a file a two but executing the rule vise versa is something I didn't see. 

I've fixed  those places manually.